src/toasts/toasts.scss

```diff
@@ -1,110 +1,90 @@
 // Toast
 
-// just for the documentation, remove before shipping
-.Toast-wrapper {
-  padding: 30px;
-}
-
 .Toast {
   display: flex;
-  background-color: white;
+  margin: $spacer-2;
   color: $black;
-  padding: $spacer-3;
+  background-color: $bg-white;
   border-radius: $border-radius;
   box-shadow: inset 0 0 0 1px $border-gray-dark, $box-shadow-medium;
-  margin: $spacer-2;
-  transition: 0.2s ease;
 
   @include breakpoint(sm) {
-    max-width: 450px;
     width: max-content;
+    max-width: 450px;
     margin: $spacer-3;
   }
 }
 
-.Toast-octicon {
-  margin-right: $spacer-2;
+.Toast-icon {
   display: flex;
   align-items: center;
+  justify-content: center;
+  width: $spacer-3 * 3;
+  flex-shrink: 0;
+  color: $text-white;
+  fill: $text-white;
+  background-color: $blue-500;
+  border-top-left-radius: inherit;
+  border-bottom-left-radius: inherit;
 }
 
-
-// Temporarily splitting this out
-// to play with static styles.
-.Toast-animated {
-  position: fixed;
-  bottom: -100px;
-  left: 0;
-  animation-name: toast;
-  animation-duration: 6s;
-  animation-delay: 1s;
-  animation-iteration-count: 1;
-  animation-timing-function: cubic-bezier(0.645, 0.045, 0.355, 1);
+.Toast-content {
+  padding: $spacer-3;
 }
 
-// Pop up animation
-@keyframes toast {
-  0%   {bottom:-100px;}
-  10%  {bottom:0px;}
-  90%  {bottom:0px;}
-  100% {bottom:-100px;}
-}
+.Toast-dismissButton {
+  max-height: 54px; // keeps button aligned to the top
+  padding: $spacer-3;
+  background-color: transparent;
+  border: 0;
 
-.Toast-block-default,
-.Toast-block-error,
-.Toast-block-warning,
-.Toast-block-success {
-  padding: 0;
-
-  .Toast-octicon {
-    color: white;
-    padding: $spacer-3;
-    margin-right: 0;
-    fill: white;
-    border-top-left-radius: inherit;
-    border-bottom-left-radius: inherit;
+  &:focus,
+  &:hover {
+    fill: $text-gray;
+    outline: none;
   }
 
-  .Toast-content {
-    padding: $spacer-3;
+  &:active {
+    fill: $gray-400;
   }
+}
 
-  .Toast-dismiss {
-    padding: $spacer-3;
-    align-self: center;
+// Modifier
 
-    &:hover {
-      cursor: pointer;
-      fill: $text-gray;
-    }
-  }
+.Toast--error .Toast-icon {
+  background-color: $red-500;
 }
 
-.Toast-block-default {
+.Toast--warning .Toast-icon {
+  fill: $gray-900;
```

Comment:
It looks like this fill color isn't showing up, we might need to select the `svg` 🤔

Reply:
Ohh.. right. This was more a hack to make the inline `<svg>` work, since the octicons styles don't get included in the code examples. 👍